graphs/helpers/badge.py

```diff
@@ -50,3 +50,53 @@ def format_coverage_precision(coverage, precision):
     precision = int(precision)
     coverage = float(coverage)
     return ("%%.%sf" % precision) % coverage
+
+
+def get_bundle_badge(bundle_size_bytes: int, precision: int):
+    bundle_size_string = format_bundle_bytes(bundle_size_bytes, precision)
+    char_width = 7  # approximate, looks good on all reasonable inputs
+    width_in_pixels = len(bundle_size_string) * char_width
+    static_width = 57  # width of static elements in the svg (text + margins)
+
+    width = static_width + width_in_pixels
+
+    return f""" <svg xmlns="http://www.w3.org/2000/svg" width="{width}" height="20">
+    <linearGradient id="CodecovBadgeGradient" x2="0" y2="100%">
+        <stop offset="0" stop-color="#bbb" stop-opacity=".1" />
+        <stop offset="1" stop-opacity=".1" />
+    </linearGradient>
+    <mask id="CodecovBadgeMask{width}px">
```

Comment:
found that ids can leak into other svgs, so trying to make the id very specific.

Reply:
Nope, figured giving it a sensical id would be better for readability. This particular element will be the same for all svgs of the same width so there won't be side effects of it leaking.